src/billing/components/Free/OrgLimits.tsx

```diff
@@ -43,19 +43,21 @@ const OrgLimits: FC = () => {
             })
         }
         if (name === 'rate') {
-          return Object.entries(value).map(([n, v]: [string, Limit]) => {
-            return (
-              <Grid.Column
-                key={n}
-                widthXS={Columns.Twelve}
-                widthSM={Columns.Six}
-                widthMD={Columns.Four}
-                widthLG={Columns.Three}
-              >
-                <OrgLimitStat name={n} value={v.maxAllowed} />
-              </Grid.Column>
-            )
-          })
+          return Object.entries(value)
+            .filter(([n]) => n !== 'queryTime')
```

Comment:
I was using the code block above as a model (line27). Additionally, when I try to remove the brackets, I get this message:
<img width="992" alt="Screen Shot 2022-02-09 at 8 48 30 AM" src="https://user-images.githubusercontent.com/64158805/153248870-65536c4c-9bf2-44f0-b963-19ed3c6d8006.png">

Reply:
@hoorayimhelping 

> ```
> .filter(([n]) => n !== 'queryTime')
>  ```

This will destructure the key name from entry and check not equals with `"queryTime"`.

> ```
> .filter(n => n !== 'queryTime')
> ```

The will compare the entire entry `[key, value]` is not equal `"queryTime"`. For our purposes, filtering out the entry with key of `"queryTime"`,  comparing  `["queryTime", 1500] !== 'queryTime'` is `true` when we'd like that entry to be `false` and filter out.